back/spec/services/json_schema_generator_service_spec.rb

```diff
@@ -202,6 +202,22 @@
           enum: %w[option1 option2]
         })
       end
+
+      it 'returns options in a random order but with other option always last if "random_option_ordering" is true' do
```

Comment:
Even thought chances of a flaky spec are low, I don't think we should do this. If we would have more specs like this we could run into flaky specs more often.

I think it's fine to manually check that random_option_ordering returns options in random order, and then sub this method in this spec and verify that the fields are returned in the same order as returned by the stub. Alternatively, perhaps we could try to use a fixed seed.